stdlib/public/runtime/MetadataLookup.cpp

```diff
@@ -1596,6 +1596,10 @@ class DecodedMetadataBuilder {
     return BuiltType();
   }
 
+  TypeLookupErrorOr<BuiltType> createMoveOnlyType(BuiltType base) const {
+    swift_unreachable("Not implemented?!");
```

Comment:
@jckarter this was the other corresponding place where I wasn't sure what to do.